src/xenia/kernel/xboxkrnl/xboxkrnl_ob.cc

```diff
@@ -78,70 +78,40 @@ DECLARE_XBOXKRNL_EXPORT1(ObLookupThreadByThreadId, kNone, kImplemented);
 dword_result_t ObReferenceObjectByHandle(dword_t handle,
                                          dword_t object_type_ptr,
                                          lpdword_t out_object_ptr) {
-  X_STATUS result = X_STATUS_SUCCESS;
+  const std::pair<XObject::Type, uint32_t> obj_type_match[] = {
```

Comment:
Could you make this a `const static` declaration? It isn't intended to be used by anyone else so `static` will make it invisible to other compilation units.

Reply:
Er, looks like I mistakenly thought that this was declared _outside_ the function.
The latter part of my last statement is false, but making this static means it won't be re-initialized whenever the function is entered.

Additionally I believe `constexpr` may also achieve the same results.